src/lib/utils/drupalClient.ts

```diff
@@ -2,19 +2,59 @@ import { DrupalClient } from 'next-drupal'
 import crossFetch from 'cross-fetch'
 import { SocksProxyAgent } from 'socks-proxy-agent'
 import { addCAs } from 'syswide-cas'
+import * as fs from 'fs'
```

Comment:
I'd like us to remove changes to this file in favor of what we'll merge in later.